patches/removed/1.18.2/0109-Option-to-remove-corrupt-tile-entities.patch

```diff
@@ -3,6 +3,7 @@ From: Zach Brown <zach.brown@destroystokyo.com>
 Date: Wed, 5 Oct 2016 16:27:36 -0500
 Subject: [PATCH] Option to remove corrupt tile entities
 
+Corrupt tile entities are never added to the world in the first place anymore.
```

Comment:
The point is that someone *tried* to add them. the option might not be needed anymore, but the check and exception are

Reply:
What check here? This patch does not touch the exception, the only thing you are losing here is the "Removing corrupt tile entity" message that is logged (which again, doesn't work because the block entity isn't added anyways).